core/src/test/java/dev/streamx/cli/ingestion/IngestionArgumentsValidationTest.java

```diff
@@ -46,8 +110,18 @@ public void shouldRejectKeylessIngestion(QuarkusMainLauncher launcher) {
     LaunchResult result = launcher.launch("unpublish", "channel");
 
     // then
-    assertThat(result.exitCode()).isNotZero();
-    assertThat(result.getErrorOutput()).contains("Missing required argument", "<key>");
+    expectError(result, """
+        Error: Missing required argument(s): <key>
+        Usage: streamx unpublish [-hV] [[--ingestion-url=<propagateIngestionUrl>]]
+                                 (<channel> <key>)
+        Send unpublication trigger
+              <channel>   Channel that message will be published to
+              <key>       Message key
+          -h, --help      Show this help message and exit.
+              --ingestion-url=<propagateIngestionUrl>
+                          Address of 'rest-ingestion-service'
+                            Default: http://localhost:8080
+          -V, --version   Print version information and exit.""");
```

Comment:
..errors in usage of unpublish commands also produce long command line errors, but shorter ones, since there is no payload and no payload definition documentation